specs/interop/derivation.md

```diff
@@ -52,7 +52,7 @@ the builder to exclude user transactions.
 
 The derivation pipeline MUST enforce that the sequencer has not included any user transactions in the batch covering
 the upgrade's activation block. If the sequencer does include any user transactions within the upgrade activation
-block, that batch, and the remaining span batch it originated from, MUST be dropped following the
+block, that batch, and the remaining span batch it originated from (if part of a span batch), MUST be dropped following the
```

Comment:
I think it's possible that the batch is a singular batch? Just covering that case here.

Reply:
This is being fixed in https://github.com/ethereum-optimism/specs/pull/694 already